src/io/flutter/logging/FlutterLogFilterPanel.java

```diff
@@ -113,6 +112,7 @@ protected void customizeCellRenderer(@NotNull JList<? extends FlutterLog.Level>
                                            boolean selected,
                                            boolean hasFocus) {
         append(value.toDisplayString());
+    logLevelComboBox.setSelectedItem(FlutterLog.Level.CONFIG);
```

Comment:
Needs formatting? Or, is this meant to live a few lines up?

Reply:
Oh, yeah.  That was dumb.

And actually, this should really be properly set in the prefs -- I didn't see the default there until just now.

Will fix.

Thanks!